React/Views/RCTWebViewManager.m

```diff
@@ -14,6 +14,8 @@
 #import "RCTWebView.h"
 #import "UIView+React.h"
 
+#if !TARGET_OS_TV
```

Comment:
These #if flags make ObjC code more complex and when we add new components people will forget to add them.